torch/fft/__init__.py

```diff
@@ -87,6 +87,98 @@
     tensor([0.+0.j, 1.+0.j, 2.+0.j, 3.+0.j])
 """)
 
+fftn = _add_docstr(_fft.fft_fftn, r"""
+fftn(input, s=None, dim=None, norm=None) -> Tensor
+
+Computes the N dimensional discrete Fourier transform of :attr:`input`.
+
+Note:
+
+    The Fourier domain representation of any real signal satisfies the
+    Hermitian property: ``X[i_1, ..., i_n] = conj(X[-i_1, ..., -i_n])``. This
+    function always returns all positive and negative frequency terms, even
+    though for real inputs, half of these values are redundant.
+    :func:`~torch.fft.rfftn` returns the more efficient "half-complex"
+    representation where only the positive frequencies of the last dimension
+    are stored explicitly.
+
+Args:
+    input (Tensor): the input tensor
+    s (Tuple[int], optional): Signal size in the transformed dimensions. If
+        given, the input will either be zero-padded or trimmed to this length
+        before computing the FFT.
+    dim (Tuple[int], optional): Dimensions to be Fourier transformed.
+        Default: all dimensions.
+    norm (str, optional): Normalization mode. For the forward transform
+        (:func:`~torch.fft.fftn`), these correspond to:
+
+        * ``"forward"`` - normalize by ``1/n``
+        * ``"backward"`` - no normalization
+        * ``"ortho"`` - normalize by ``1/sqrt(n)`` (making the FFT orthonormal)
+
+        Where ``n = prod(s)`` is the logical FFT size.
```

Comment:
"s" may not be specified, so `n = prod(s)` will have to change.

Reply:
How about if I add to the docs:
> Default: `s = [input.size(d) for d in dim]`

Reply:
OK.